mise.toml

```diff
@@ -0,0 +1,29 @@
+[tools]
+# Project overrides (formerly .pulumi.version)
```

Comment:
Saying "formerly" .. are we removing the .pulumi.version .. or are we still using that for the makefile dependency?